definitions/tools/concordance.cwl

```diff
@@ -1,48 +1,64 @@
+#!/usr/bin/env cwl-runner
+
 cwlVersion: v1.0
 class: CommandLineTool
-label: "Concordance tool"
-baseCommand: ["python", "/opt/concordance/newConcordance.py"]
+label: "Concordance checking between Tumor and Normal BAM"
+baseCommand: ["somalier"]
```

Comment:
It might be nice to use a fully-qualified path here.

Reply:
its a binary. so if we just use the docker container shouldn't it run properly?

Reply:
Theoretically, yes it should! When running Docker there is the option to override environment variables, so it cannot be guaranteed that the `PATH` will be set appropriately.  Some tools require the `PATH` to be corrected. If this one is self-contained, then pointing to the full path inside the container will help eliminate some potential errors.  Adding it isn't required; it just might be nice 😄

Reply:
How do I find out the path to that binary?

Reply:
You can go inside the docker container to find out.  For instance:
```
$ LSF_DOCKER_PRESERVE_ENVIRONMENT=false docker-interactive brentp/somalier
Job <7938596> is submitted to queue <docker-interactive>.
<<Waiting for dispatch ...>>
<<Starting on blade18-1-15.gsc.wustl.edu>>
Using default tag: latest
latest: Pulling from brentp/somalier
Digest: sha256:eefd6bd95826ce287a1d9201cfa01feaa6982213077c174caa8fba6546daf43f
Status: Image is up to date for brentp/somalier:latest
/usr/bin/id: cannot find name for user ID 12347
/usr/bin/id: cannot find name for group ID 10001
/usr/bin/id: cannot find name for user ID 12347
bash-4.1$ which somalier
~/usr/bin/somalier
bash-4.1$ ls -l /usr/bin/somalier 
-rwxr-xr-x 1 root root 349690 Apr 19 20:32 /usr/bin/somalier
```
so here it's at `/usr/bin/somalier`.

Reply:
I did the changed